src/ralph/converters/xapi/server.py

```diff
@@ -0,0 +1,40 @@
+"""
+Server event Xapi Converter
+"""
+from tincan import Activity, ActivityDefinition, LanguageMap
+
+from ralph.schemas.edx.server import ServerEventSchema
+
+from .base import BaseXapiConverter
+from .constants import (
+    EN,
+    HOME_PAGE,
+    PAGE,
+    VIEWED,
+    XAPI_ACTIVITY_PAGE,
+    XAPI_EXTENSION_REQUEST,
+    XAPI_VERB_VIEWED,
+)
+
+
+class ServerXapiConverter(ServerEventSchema, BaseXapiConverter):
+    """Converts a common edx server event to Xapi
+    See ServerEventSchema for info about the Edx server event
+    Example Statement: John viewed https://www.fun-mooc.fr/ Web page
+    """
+
+    def get_verb_kwargs(self):  # pylint: disable=no-self-use
+        """Returns tincan.Verb kwargs for the xapi_event"""
+        return {"id": XAPI_VERB_VIEWED, "display": LanguageMap({EN: VIEWED})}
```

Comment:
May be http://id.tincanapi.com/verb/viewed is not the best option to describe the user action (verb) ?
(as edx logs server events for almost any kind of request - example the requests for the translations script: /i18n.js)

Other options could be:
http://adlnet.gov/expapi/verbs/launched
https://w3id.org/xapi/adb/verbs/requested
https://w3id.org/xapi/adb/verbs/demanded

WDYT?